pygt3x/activity_payload.py

```diff
@@ -21,6 +21,8 @@ def unpack_bitpack_acceleration(source: bytes):
 
     """
     data = np.frombuffer(source, dtype=np.uint8)
+    if (data.shape[0] % 3) == 1:
+        data = data[:-4]
```

Comment:
I see this makes it possible to reshape 81647788 (27215929,3) as in #39...
But, why we don't drop only the last row (`data=data[:-1]`)?
Can `(data.shape[0] % 3) == 2` in some other case?

Reply:
I haven't seen another case yet. This works in this case.

Reply:
Fixed to read one more row of data. I also updated the documentation.